src/fastpm.c

```diff
@@ -626,7 +643,7 @@ prepare_ic(FastPMSolver * fastpm, Parameters * prr, MPI_Comm comm)
 }
 
 static void 
-prepare_ncdm(FastPMSolver * fastpm, Parameters * prr, MPI_Comm comm) 
+prepare_ncdm(FastPMSolver * fastpm, Parameters * prr, FastPMConfig * config, MPI_Comm comm) 
```

Comment:
Do you use config? I didn't see it in the function. You can either get the config value from CONF(prr->lua), or from member of fastpm->config.

Reply:
I use it when shifting, e.g. for config->USE_SHIFT, but yes, I'll replace with fastpm->config